src/coreComponents/dataRepository/GroupContext.cpp

```diff
@@ -58,7 +66,15 @@ WrapperContext::WrapperContext( WrapperBase & wrapper ):
 
 string WrapperContext::toString() const
 {
-  return m_group.getDataContext().toString( m_typeName );
+  ToStringInfo const info = m_group.getDataContext().getToStringInfo();
+  if( info.hasInputFileInfo())
+  {
+    return GEOS_FMT( "{}, attribute {}", m_group.getDataContext().toString(), m_typeName );
+  }
+  else
+  {
+    return GEOS_FMT( "{}->{}", m_group.getDataContext().toString(), m_typeName );
```

Comment:
```suggestion
  string const tmp = info.hasInputFileInfo() ? "{}, attribute {}" : "{}->{}";
  return GEOS_FMT( tmp, m_group.getDataContext().toString(), m_typeName );
```
?

Reply:
I changed the output of this method to uniformize it with the `DataFileContext::toString()` output, but I used a ternary as you suggested to shorten the code.